content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/dotnet-core.md

```diff
@@ -88,7 +88,7 @@ You can install the Datadog .NET Tracer machine-wide so that all services on the
 
 To install the .NET Tracer machine-wide:
 
-1. Download the [.NET Tracer MSI installer][1]. Select the MSI installer for the architecture that matches the operating system (x64 or x86).
+1. Download the [.NET Tracer MSI installer][1].
```

Comment:
Same note as on the .NET Framework page: Should we outright recommend the x64 installer?

Reply:
Yeah, good question 🤔 In 3.0 we _only_ have the x64 installer, so there's nothing to choose. But if they install the earlier version then they would have the option. So in that case, yeah, maybe it makes sense. Something like this?

```suggestion
1. Download the [.NET Tracer MSI installer][1]. Use the x64 MSI installer if you are running 64-bit Windows; this can instrument both 64-bit and 32-bit applications. Only choose the x86 installer if you are running 32-bit Windows. Starting with v3.0.0, only the x64 installer is provided, as we do not support 32-bit operating systems.
```

Reply:
That looks great!